### DISCUSSION TITLE
How can i make branch and PR to main?

### DISCUSSION THREAD

**Discussion by pranahonk** [mastodon/mastodon#22321] [General]
btw i want to contribute to mastodon. but i cannot make a branch from mastodon. may i know how to make branch from mastodon and i want to Pull Request

**Comment by Gargron** [comment]
You need to fork it to create branches and submit PRs.

**Comment by pranahonk** [comment]
is there any bug on the web that I need to fix? btw how do I want to see the list bugs in mastodon @Gargron

**Reply by YJSoft** [reply]
https://github.com/mastodon/mastodon/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug

Or you can install extension [Refined Github](https://github.com/refined-github/refined-github) to add "Bugs" tab to filter only labeled with "bug"
